src/lib/components/chat/Settings/General.svelte

```diff
@@ -18,6 +18,41 @@
 
 	let showAdvanced = false;
 
+	window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change', e => {
```

Comment:
Haven't looked into much yet, but we might want to add this code to `app.html` to prevent FOUC

Reply:
Good point I'll take a look